ui/src/styles.css

```diff
@@ -269,3 +269,19 @@ iframe#launcher {
   border-right: 1px solid #c8c8c8;
   border-top: 1px solid #c8c8c8;
 }
+
+.p-paginator {
+  height: 2rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.p-paginator .p-dropdown.p-component.p-inputwrapper-filled {
+  height: 1.5rem;
+}
+
+body .p-paginator .p-paginator-pages {
+  height: auto;
+}
+
```

Comment:
Are these styles used in Datatable style?

Reply:
Yeah, these are the classes that PrimeReact uses to style its DataTable component.